conduct.md

```diff
@@ -120,15 +120,15 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Openness. (vulnerability)
```

Comment:
Can you please elaborate on this point. What do you mean by that? How does it apply in the context of the class interactions and conduct?